bimsb/packages/bioinformatics-nonfree.scm

```diff
@@ -1882,3 +1883,72 @@ MACE is a bioinformatics tool dedicated to analyze ChIP-exo data.")
     ;; others say "free for all use", others say "GPL version 1.3" (?), yet
     ;; others say "Artistic license".
     (license license:gpl2+)))
+
+;; Although this program is released under the GPL it depends on
+;; ViennaRNA and bowtie1, which are non-free software.
+(define-public mirdeep2
+  (package
+    (name "mirdeep2")
+    (version "0.0.92")
+    (source (origin
+              (method git-fetch)
+              (uri (git-reference
+                   (url "https://github.com/rajewsky-lab/mirdeep2.git")
+                   (commit (string-append "v" version))))
+              (file-name (string-append name "-" version "-checkout"))
+              (sha256
+               (base32
+                "1gasx6vb38n4vbjbn5s7ch2rpflgf2crdd8mdp2xm571w2j01wmm"))))
+    (build-system gnu-build-system)
+    (arguments
+     `(#:phases
+       (modify-phases %standard-phases
+         (delete 'configure)
+         (delete 'build)
+         (delete 'check)
+         (replace 'install
+           (lambda* (#:key inputs outputs #:allow-other-keys)
+             (let ((target (string-append (assoc-ref outputs "out")
+                                          "/bin")))
+               (with-directory-excursion "src"
+                 (copy-recursively "." target))
+               (copy-file "Rfam_for_miRDeep.fa"
+                          (string-append  target "/../Rfam_for_miRDeep.fa"))
```

Comment:
@rekado: This is the second file I was mentioning.

Reply:
Yeah, I don't like this.  I'd prefer to place these files in `$out/share/mirdeep2/` and patch the other files.  `share/<package>` is the canonical place to put "miscellaneous" package data.

Reply:
Sure - I see why you don't like this. Any pointers as to what is the proper way to patch the sources?